src/monster.cpp

```diff
@@ -1441,6 +1441,17 @@ bool monster::is_immune_damage( const damage_type dt ) const
     }
 }
 
+void monster::make_bleed( const effect_source &source, const bodypart_id &bp,
+                          time_duration duration, int intensity, bool permanent, bool force, bool defferred )
+{
+    duration = ( duration * type->bleed_rate ) / 100;
```

Comment:
Perhaps an early exit?
```suggestion
    if( type->bleed_rate == 0 ) {
        return;
    }

    duration = ( duration * type->bleed_rate ) / 100;
```